deployment/hasura/migrations/Aerie/3_sequence_parcel/up.sql

```diff
@@ -0,0 +1,378 @@
+  owner text,
+  updated_at timestamptz not null default now(),
+  updated_by text,
+
+  constraint sequence_adaptation_synthetic_key
+    primary key (id),
+  foreign key (owner)
+    references permissions.users
+    on update cascade
+    on delete set null,
+  foreign key (updated_by)
+    references permissions.users
+    on update cascade
+    on delete set null
+);
+
+comment on table sequencing.sequence_adaptation is e''
+  'A custom adaptation used to overwrite variable and linting rules for the sequence editor';
+comment on column sequencing.sequence_adaptation.adaptation is e''
+  'The sequencing adaptation code.';
+
+create trigger set_timestamp
+  before update on sequencing.sequence_adaptation
+  for each row
+execute function util_functions.set_updated_at();
+
+
+--------------------------------------
+---- Create the Parcel Table
+--------------------------------------
+create table sequencing.parcel (
+ id integer generated always as identity,
+
+ name text not null,
+
+ command_dictionary_id integer not null,
+ channel_dictionary_id integer default null,
+ sequence_adaptation_id integer default null,
+
+ created_at timestamptz not null default now(),
+ owner text,
+ updated_at timestamptz not null default now(),
+ update_by text,
+
+ constraint parcel_synthetic_key
+   primary key (id),
+
+ foreign key (channel_dictionary_id)
+   references sequencing.channel_dictionary (id)
+   on delete set null,
+ foreign key (command_dictionary_id)
+   references sequencing.command_dictionary (id)
+   on delete cascade,
+ foreign key (sequence_adaptation_id)
+   references sequencing.sequence_adaptation (id)
+   on delete set null,
+ foreign key (owner)
+   references permissions.users
+   on update cascade
+   on delete set null,
+  foreign key (update_by)
+   references permissions.users
+   on update cascade
+   on delete set null
+);
+
+comment on table sequencing.parcel is e''
+  'A bundled containing dictionaries and a sequence adaptation file.';
+comment on column sequencing.parcel.id is e''
+  'The synthetic identifier for this parcel.';
+comment on column sequencing.parcel.name is e''
+  'The name of the parcel.';
+comment on column sequencing.parcel.channel_dictionary_id is e''
+  'The identifier for the channel dictionary.';
+comment on column sequencing.parcel.command_dictionary_id is e''
+  'The identifier for the command dictionary.';
+comment on column sequencing.parcel.sequence_adaptation_id is e''
+  'The identifier for the sequencing adaptation file.';
+
+create trigger set_timestamp
+  before update on sequencing.parcel
+  for each row
+execute function util_functions.set_updated_at();
+
+--- populate table with data
+insert into sequencing.parcel (name, channel_dictionary_id, command_dictionary_id, sequence_adaptation_id, created_at, updated_at, owner)
+select
+    'Parcel ' || cd.id as name,
+    null,
+    cd.id,
+    null,
+    now(),
+    now(),
+    'Aerie Legacy'
+from sequencing.command_dictionary cd;
```

Comment:
Nonblocking: this insert can be simplified. 
Blocking: "updated_by" is missing from this insert.
```suggestion
insert into sequencing.parcel (name, command_dictionary_id, owner, updated_by)
select
    'Parcel ' || cd.id as name,
    cd.id,
    'Aerie Legacy',
    'Aerie Legacy'
from sequencing.command_dictionary cd;
```

Reply:
Thank you for the modified SQL statement. I updated the `up.sql `